h/static/scripts/test/root-thread-test.js

```diff
@@ -212,6 +212,19 @@ describe('rootThread', function () {
       assert.isTrue(threadFilterFn(fakeBuildThread));
     });
 
+    it('generates a thread filter function to match orphans', function () {
```

Comment:
From the phrasing of these test names, it is a little difficult to understand at first how they relate to what the user sees.

I would suggest something like "filter matches only orphans when Orphans tab is selected" and similar for the other related tests.